docs/extending.rst

```diff
@@ -113,3 +113,20 @@ You will need to customize the following templates:
     * ``django_comments_xtd/email_confirmation_request.{txt|html}`` to add the new fields to the confirmation request, if it was necessary. This demo overrides them to include the ``title`` field in the mail.
     * ``django_comments_xtd/comments_tree.html`` to show the new field when displaying the comments. If your project doesn't allow nested comments you can use either this template or `comments/list.html``.
     * ``django_comments_xtd/reply.html`` to show the new field when displaying the comment the user is replying to.
+
+
+Modifying comments with code
+====================
+Here's an example of how to access the underlying model storing your comments
+
+    from django_comments_xtd.models import XtdComment
+    from django.contrib.contenttypes.models import ContentType
+    
+    def make_comments_nonpublic(model_instance):
```

Comment:
This function doesn't make comments non-public. It does only update the `followup` but exclude some email address from the update. Why the name of the function doesn't match what it does? Could you elaborate the goal of this new section of the documentation? I'm a bit lost here.

Reply:
you are totally right. I've renamed (was relevant for my usage)